modules/lakeformation/variable.tf

```diff
@@ -0,0 +1,14 @@
+variable "emr_role" {
```

Comment:
Could we rename this to `emr_instance_profile_role` to make it clear which EMR role is needed here?